### PR TITLE
Dockerfile: changing to non-privileged port

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM ubuntu:14.04
```
Comment (bryanlatten) [inline Dockerfile]: rolling back, wasn't helpful downstream

**Comment by anshupande** [comment]
👍

File: container/root/run.d/99-nginx.sh [hunk 2]
```diff
@@ -5,9 +5,6 @@ if [ $CONTAINER_ROLE == 'web' ]
 then
   echo '[run] enabling web server'
 
-  # Unfortunately, until Dockerhub supports this operation...it has to be done here
-  setcap cap_net_bind_service=+ep /usr/sbin/nginx
```
Comment (bryanlatten) [inline container/root/run.d/99-nginx.sh]: dockerhub couldn't run this command, neither can docker 1.11 by default

Comment (bossjones) [inline container/root/run.d/99-nginx.sh]: Is this just an option that needs to run via `root` ?

File: Dockerfile [hunk 3]
```diff
@@ -48,7 +48,8 @@ COPY ./container/root /
 RUN tar xzf /tmp/s6-overlay-amd64.tar.gz -C / && \
     rm /tmp/s6-overlay-amd64.tar.gz
 
-EXPOSE 80
+# Using a non-privileged port to prevent having to use setcap internally
+EXPOSE 8080
```
Comment (bossjones) [inline Dockerfile]: Do we need to modify this in Flight Director to make sure it matches this new port? across whichever apps are running from this parent container?

File: container/root/run.d/99-nginx.sh (hunk 2 above)
Comment (bryanlatten) [inline container/root/run.d/99-nginx.sh]: its something that allows you to run on privileged port *without* root on nginx

**Comment by Attamusc** [comment]
This _might_ require updates to consumers using tugboat, but worth checking just to make sure it grabs this exposed port correctly

File: Dockerfile (hunk 3 above)
Comment (bryanlatten) [inline Dockerfile]: yes - when we roll it out. I will make this a major change (5.0)

**Comment by bryanlatten** [comment]
@Attamusc no tugboat change needed. it picks up everything!

**Comment by Attamusc** [comment]
woohoo!

File: Dockerfile (hunk 3 above)
Comment (bossjones) [inline Dockerfile]: Excellent. Just checking.

File: container/root/run.d/99-nginx.sh (hunk 2 above)
Comment (bossjones) [inline container/root/run.d/99-nginx.sh]: Got it, cool

**Comment by bossjones** [comment]
As long as roll out to existing containers is done carefully and there is no fear of this being auto rolled out. w/ this `80->8080`, lgtm ! 👍

**Pull request merged**